docs/background/plot_00_conceptual_intro.py

```diff
@@ -188,3 +194,7 @@
 #     `score` method, passing the predictors and the counts. The method first
 #     computes the rate $\lambda(t)$ using (2) and then the likelihood using
 #     (4). This method is used under the hood during optimization.
+
+# %%
+### More general GLMs
+# So far, we have focused on the relatively simple LNP model of spike generation, which is a special case of a GLM. The LNP model has some known shortcomings (Pillow et al 2008). For instance, LNP ignores things like refactory periods and other history-dependent features of spiking in a neuron. As we will show in other demos, such _spike history filters_ can be built into GLMs to give more accurate results. We will also show how, if you have recordings from a large _population_ of neurons simultaneously, you can build connections between the neurons into the GLM in the form of _coupling filters_. This can help answer the degree to which activity is driven primarily by the input X, or by network influences in the population. Some researcheers have used cascades of GLMs to model dendritic computation in individual neurons (Ujfalussy et al 2018). 
```

Comment:
```suggestion
# So far, we have focused on the relatively simple LNP model of spike generation, which is a special case of a GLM. The LNP model has some known shortcomings (Pillow et al 2008). For instance, LNP ignores things like refactory periods and other history-dependent features of spiking in a neuron. As we will show in other demos, such _spike history filters_ can be built into GLMs to give more accurate results. We will also show how, if you have recordings from a large _population_ of neurons simultaneously, you can build connections between the neurons into the GLM in the form of _coupling filters_. This can help answer the degree to which activity is driven primarily by the input X, or by network influences in the population.
```

I checked the ref, the model is more similar to a standard RNN than a GLM. Citing: "The error is a non-convex function of the parameters with multiple local minima. ". I think this would be misleading for an inexperienced audiece

Reply:
It's basically an NN with a hierarchical structure to its connection